Handler/Feed.hs

```diff
@@ -24,10 +24,10 @@ feedFromComments (Entity siteId site) comments = do
     render <- getUrlRender
 
     rssFeedText Feed
-        { feedAuthor      = siteRssAuthor site
-        , feedTitle       = siteRssTitle site
-        , feedDescription = siteRssDescription site
-        , feedLanguage    = siteRssLanguage site
+        { feedAuthor      = siteName site
+        , feedTitle       = "Comments on " <> siteName site
+        , feedDescription = toHtml $ "Comments on " <> siteName site
```

Comment:
I'll double-check, I'm pretty sure you have to use a `prescapedToHtml` or something to avoid escaping.